components/counter/index.tsx

```diff
@@ -139,38 +147,49 @@ const CircularProgressBar = (props) => {
 	);
 };
 
-/**
- * Counter
- *
- * @description display character count and limit.
- *
- * @returns <Counter />
- */
-const Counter = forwardRef((props, ref) => {
-	const { count, limit } = props;
-	const percentage = (count / limit) * 100;
-	return (
-		<StyledComponentContext cacheKey="tenup-component-counter">
-			<StyledCounter
-				className={cx('tenup--block-components__character-count', {
-					'is-over-limit': count > limit,
-				})}
-				{...props}
-				ref={ref}
-			>
-				<div className="tenup--block-components__character-count__label">
-					<span className="tenup--block-components__character-count__count">{count}</span>{' '}
-					/{' '}
-					<span className="tenup--block-components__character-count__limit">{limit}</span>
-				</div>
-				<CircularProgressBar percentage={percentage} />
-			</StyledCounter>
-		</StyledComponentContext>
-	);
-});
-
-CircularProgressBar.propTypes = {
-	percentage: PropTypes.number.isRequired,
-};
+interface CounterProps {
+	/**
+	 * Current count.
+	 */
+	count: number;
+
+	/**
+	 * Max limit.
+	 */
+	limit: number;
+
+	/**
+	 * Rest of the props.
+	 */
+	[key: string]: unknown;
```

Comment:
❤️ thanks for adding `unknown` this time!

Reply:
Yes, I remembered this time! 😄